### PR TITLE
add lacinia & artemis deps, begin graphql endpoint

### LINKED ISSUE #73: feat: add GraphQL libraries for CLJ and CLJS
Recommended libraries for GraphQL support in Clojure(Script):

- frontend: http://docs.workframe.com/artemis/current/index.html
- backend: https://github.com/walmartlabs/lacinia

### PR DESCRIPTION
**Pull request by eemshi**
closes #73

### CODE REVIEW

File: src/clj/cwbn/routes/services.clj [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(ns cwbn.routes.services
+  (:require [ring.util.http-response :refer :all]
+            [compojure.api.sweet :refer :all]
+            [cwbn.routes.services.graphql :as graphql]
+            [schema.core :as s]))
+
+(defapi service-routes
+  (context "/api" []
+    :tags ["thingie"]
```
Comment (dviramontes) [inline src/clj/cwbn/routes/services.clj]: can we include a description here or remove the :tag ? 🙏

File: project.clj [hunk 2]
```diff
@@ -26,6 +29,7 @@
                  [org.webjars.bower/tether "1.4.3"]
                  [org.webjars/bootstrap "4.1.0"]
                  [org.webjars/font-awesome "5.0.10"]
+                 [org.webjars/webjars-locator "0.34"]
```
Comment (dviramontes) [inline project.clj]: not sure we need this dependency...

Comment (eemshi) [reply project.clj]: thanks!

File: src/clj/cwbn/routes/services.clj (hunk 1 above)
Comment (eemshi) [reply src/clj/cwbn/routes/services.clj]: 👍 added a comment

**Pull request merged**